### PR TITLE
#73 document browser compatibility

### PR DESCRIPTION
**Pull request by srdmathur**
I have hereby documented the browser compatibility information. Kindly review.
I hope it will be as per expectation. 
If anything you feel missing, please let me know. I will try to improve.

### CODE REVIEW

File: docs/Compatibility.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Compatibility
+---
+
+# Compatibility
+
+## Browser Support 
+
+Smooth-UI rely on [browsers supported by Styled Components](https://www.styled-components.com/docs/faqs#which-browsers-are-supported) or browsers supported by emotion (probably the same).
```
Comment (neoziro) [inline docs/Compatibility.mdx]: Smooth UI relies

File: docs/Compatibility.mdx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+As Smooth-UI is using v4.x of Styled Components, which is using React v16.3+, so it will support same browsers as the version of React and Styled Components.
```
Comment (neoziro) [inline docs/Compatibility.mdx]: I think this paragraph brings nothing to the user.

Comment (neoziro) [reply docs/Compatibility.mdx]: Can you remove it?

Comment (srdmathur) [reply docs/Compatibility.mdx]: Sure

File: docs/Compatibility.mdx (hunk 1 above)
Comment (srdmathur) [reply docs/Compatibility.mdx]: I will correct this

**Comment by neoziro** [comment]
Good for me thank you!

**Pull request merged**